gamestonk_terminal/stocks/fundamental_analysis/fa_controller.py

```diff
@@ -721,6 +721,10 @@ def call_fraud(self, other_args: List[str]):
                 " higher probability of default. One of the criticisms that Zmijewski made was that"
                 " other bankruptcy scoring models oversampled distressed firms and favored situations"
                 " with more complete data.[Source: YCharts]"
+                "McKee-score:\n------------------------------------------------\n"
+                "The McKee Score is a bankruptcy model used to predict a firm's bankruptcy in one year"
+                "It looks at a companie's size, profitability, and liquidity to determine the probability."
+                "This model is 80% accurate in predicting bankruptcy."
```

Comment:
if you gonna claim something like 80% accuracy, I need a source (probably In the Hugo suffices)

Reply:
Research paper in the hugo.